src/common/containers/build-details.js

```diff
@@ -80,10 +78,15 @@ class BuildDetails extends Component {
             </div>
           </div>
         }
-        <HelpInstallSnap
-          headline='To test the latest successful build on your PC or cloud instance:'
-          name='foo'
-        />
+        { isFetching &&
+          <div className={styles.spinner}><Spinner /></div>
+        }
+        { snap &&
+          <HelpInstallSnap
+            headline='To test the latest successful build on your PC or cloud instance:'
+            name={ snap.store_name }
```

Comment:
We may think about transforming data from API on reducer level - to store only data we need for the app under names we control in BSI.
That would help in such case, and also it will allow us to avoid mixing camelCase that we use as standard with snake_case that we get from LP APIs.